src/components/Reservation.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchHotels } from '../Redux/Actions/hotels';
+import localStorageActions from '../utils/localStorage';
+
+const Reservation = (props) => {
+  const {
+    reason, duration, startDay, endDay, id, hotelID,
+  } = props;
+
+  const dispatch = useDispatch();
+
+  const Hotels = useSelector((state) => state.hotels);
+  useEffect(() => {
+    dispatch(fetchHotels());
+  }, [dispatch]);
+
+  const userObject = localStorageActions.getUser();
+
+  return (
+    <div className="reservation" id={id}>
+      <p>
+        Reason for staying:
+        {reason}
+      </p>
+      <p>
+        Duration:
+        {duration}
+        <small>{duration === '1' ? 'day' : 'days'}</small>
```

Comment:
- [ ] In order to separate the `days` from the number, you could add a space on each like this::

```HTML
<small>{duration === '1' ? ' day' : ' days'}</small>
```

Notice the space before " day" and " days".

Result:

![image](https://user-images.githubusercontent.com/57421823/193183342-dbb58074-5daa-4bcf-b6fc-fca307ef3866.png)